tests/utils.py

```diff
@@ -166,6 +169,9 @@ def delete_permission(self, permission):
     def delete_resource(self, res_id):
         pass
 
+    def resync(self):
+        pass
```

Comment:
Use `NotImplementedError` insatead of `pass` +  FIXME comment.

Reply:
I used `pass` specifically here (and the other mentioned lined below) instead of `NotImplementedError`, since this is just a Mocking class. This function might never be implemented even if the actual method found in the real `Magpie` class gets eventually implemented.

This is similar to the rest of the Mock class. Other functions such as `user_created`, `user_deleted`, etc. are not implemented and only have a pass, since they don't need to have any content for the current tests.

I would keep the `pass`, as I don't see a reason to raise an exception here.

Reply:
I see it more as a safeguard. If the mock method gets called by mistake somehow, the explicitly raised `NotImplementedError` will indicate it clearly. The message of that raised error could also indicate something such as "This mock method implementation should not be called" to make the intention even more verbose.

Reply:
I get your idea, but I'm not sure if it applies nicely here. I works if I add `NotImplementedError` to the `resync` method only, but there are other methods in the Mock class that use the same `pass` behavior. I think it wouldn't be coherent to use different behaviors (`pass` or `NotImplementedError`) for similar methods, which are either implemented, not implemented yet or will never be implemented on the actual `Handler` classes.

Also, if they all use the `NotImplementedError`, I think it will break multiple tests that use Mock classes. Some tests call API endpoints, which can use Mock handlers, and which would now break, having errors during the API call. Other tests use Mock handlers for testing celery request tasks.

I could always try to change those tests, but I am not sure if it is worth the effort?

Reply:
In places that use `pass` as a "don't care" or some kind of override for testing purposes, it is ok to leave it as is to skip the resync that "should" normally be called. 

The issue was more for places that use `pass` instead of `NotImplementedError` for cases that are "pending implementation".

In this case, seems to be ok.